hvm/linkManager/set_bin_link.go

```diff
@@ -0,0 +1,72 @@
+package linkmanager
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func SetBinLink(org, project, version string) error {
+
+	user, err := user.Current()
+
+	if err != nil {
+		return errors.Wrapf(err, "Error getting current user information")
+	}
+
+	home := user.HomeDir
+	versionsPath := path.Join(home, ".czi", "versions", org, project, version)
+	binPath := path.Join(home, ".czi", "bin")
+
+	os.MkdirAll(binPath, 0755)
+
+	if _, err := os.Stat(versionsPath); os.IsNotExist(err) {
+		fmt.Println("Requested version is not installed.")
+		return err
```

Comment:
```suggestion
	if _, err := os.Stat(versionsPath); os.IsNotExist(err) {
		return errors.Wrap(err, "requested version is not installed")
```

Reply:
bubble up all the errors to one place and print them at the root of the application